workers/unity/Packages/com.improbable.gdk.mobile/Utility/LaunchArguments.cs

```diff
@@ -36,7 +38,30 @@ public static class LaunchArguments
                 Debug.LogException(e);
             }
 
-            return new Dictionary<string, string>();
+            return null;
+        }
+
+        public static Dictionary<string, string> GetiOSArguments()
+        {
+            if (Application.isEditor)
+            {
+                return new Dictionary<string, string>();
+            }
+
+            try
+            {
+                var argumentsEnvVar = System.Environment.GetEnvironmentVariable(LaunchArgumentsEnvKey);
```

Comment:
just checking (after I had all these weird issues with environment variables): does that work properly?

Reply:
yeah, this works as expected apparently as we pass it to the process directly